client/landing/stepper/declarative-flow/internals/steps-repository/components/flow-card/style.scss

```
@@ -34,9 +34,14 @@ $blueberry-color: #3858e9;
 		padding: 2px 1px 0 0;
 	}
 
+	.flow-question__title-wrapper {
+		display: flex;
+	}
+
 	.flow-question__heading {
 		font-weight: 500;
 		font-size: $font-body-large;
+		margin-right: 8px;
```

```suggestion
		display: flex;
		align-items: center;
		gap: 0.5em;
```
This would also better align the badge vertically.